app/authentication_views.py

```diff
@@ -57,17 +60,16 @@ def register():
             f"Handling '/register' route, registering new user {username} is finished successfully")
         return "success"
 
-    else:
-        logger.info(
-            "Handling '/register' route, user is not authenticated and from not submitted, rendering register.html")
-        return render_template("forms/register.html")
+    logger.info(
+        "Handling '/register' route, user is not authenticated and from not submitted, rendering register.html")
```

Comment:
'form' not 'from'
'user is not authenticated' so
'form is not submitted'

Reply:
done